README.md

```diff
@@ -43,3 +44,11 @@ To run tests, open /test.html in a browser. If you have phantomjs installed, you
 You can find the installation instructions for phantomjs in its official [build documentation](http://phantomjs.org/build.html). For Ubuntu/debian based system you can follow [these instructions](https://gist.github.com/julionc/7476620) or use the script mentioned there.
 
 To add new tests, edit the `*_spec.js` files in `/spec/javascripts/`.
+
+## Options
+
+| Methods | Use | Usage (Example) |
+|---------|-----|-----------------|
+|getLat   | Used to get the current latitude of the center of the map. | blurredLocation.getLat() //This would return the value in numerics |
```

Comment:
Yeah I've added that. I think this looks good to merge now. What do you say? @jywarren